index.html

```diff
@@ -18,33 +18,27 @@ <h1>Smol Beans</h1>
       <h2 class="sub-title">Rat Rescue</h2>
       <h3 class="sub-sub-title">Denver & Colorado Springs</h3>
     </div>
-        <!-- <form class="search">
-          <input type="search" placeholder="Search..." name="search" id="filterNameIngInput"/>
-          <button type="submit" value="search" id="submitNameIng">search</button>
-        </form> -->
         <img class="hamburger" src="./assets/hamburger.png" alt="Hamburger menu">
         <nav>
-          <!-- <img class="hamburger" src="./assets/hamburger.png" alt="Hamburger menu"> -->
-          <button>About Us</button>
-          <button>Adoptables</button>
-          <button>Donate</button>
-          <!-- <button>Calendar</button> -->
-          <img class="calendar" src="./assets/calendar.png" alt="Calendar">
+          <button class="about-us">About Us</button>
+          <button class="adoptables">Adoptables</button>
+          <button class="donate">Donate</button>
+          <!-- <img class="calendar" src="./assets/calendar.png" alt="Calendar"> -->
         </nav>
     </header>
     <main>
       <h2>Adopt a friend!</h2>
       <section class="adoption-section">
       <section class="adoption-card">
-        <div class="overlay">
+        <div class="container">
           <div class="image-info">
             <div class="views">
               <p>views</p>
               <p class="view-num">15</p>
             </div>
               <img class="icon" src="./assets/heart.png" alt="Heart Icon"/>
           </div>
-          <img class="rat-photo" src="./assets/belinda.jpeg" alt="A photo of Belinda the rat">
+            <img class="rat-photo" src="./assets/belinda.jpeg" alt="A photo of Belinda the rat">
```

Comment:
Generally, with alt text, you don't want to include phrases like "a photo of" or "an image of" because that's already implied. You can just say "Belinda the rat".